src/modules/RootPart.rb

```diff
@@ -1240,6 +1240,25 @@ def MountFSTab(fstab, message)
         )
       end
 
+      efivars_path = "/sys/firmware/efi/efivars"
+      if ::File.exist?(efivars_path)
+        if MountPartition(efivars_path, "efivarfs", "efivarfs") == nil
+          AddMountedPartition(
+            { :type => "mount", :device => "efivarfs", :mntpt => efivars_path }
+          )
+        end
+      end
+
+      if SCR.Execute(
+          path(".target.mount"),
+          ["/run", Installation.destdir + "/run", Installation.mountlog],
+          "--bind"
+         )
```

Comment:
Please explain why that MountPartition method cannot be used here.

Reply:
Comment added:
```rb
      # MountPartition does not work here
      # because it turns --bind into -o --bind
```

Reply:
and any clue why `-o --bind` does not work?

Reply:
Because it turns `--bind` from an option for the `mount` command into a "mount option", that is, an option for a single mount, like `noatime`. Before it can fail as a non-existing mount option, the command discovers that the "device" it is trying to work on is in fact a directory so it says "mount: /mnt/run: /run is not a block device."